internal/cli/up.go

```diff
@@ -147,7 +147,7 @@ func (c *upCmd) run(ctx context.Context, args []string) error {
 
 	g.Go(func() error {
 		defer cancel()
-		return upper.Start(ctx, args, c.watch, triggerMode, c.fileName, c.hud)
```

Comment:
it might be easier to dependency inject this with wire, so you don't need to wire it all up yourself?

Reply:
huhhh wiring this straight into `NewStore()` is preferable? idk how to get it onto the store via wire otherwise...